media-sound/beets/beets-1.4.9.ebuild

```diff
@@ -0,0 +1,139 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PV=${PV/_beta/-beta.}
+MY_P=${PN}-${MY_PV}
+
+PYTHON_COMPAT=( python3_{5,6} )
```

Comment:
Looking at the deps I don't believe it'll work, but could/did you check with repoman if python3.7 is supported? Please test it if yes :)

Reply:
It explodes, unfortunately :P